UPGRADE.md

```diff
@@ -0,0 +1,114 @@
+# Upgrade to Ceramic One from Kubo
+
+This document describes how to upgrade an existing Ceramic node using Kubo to run with `ceramic-one` while preserving existing data.
+
+## Overview
+
+Follow these steps for basic upgrade of a Ceramic node with some downtime.
+
+1. Shutdown existing `js-ceramic` node.
+2. Migrate data from IPFS blocks into Sqlite database file `ceramic-one` can consume.
+3. Start `ceramic-one` daemon.
+4. Configure `js-ceramic` to run against `ceramic-one`.
+5. Start `js-ceramic` node and validate.
+
+If you require a zero downtime upgrade see the section below that expands the basic steps for a near zero downtime upgrade.
+
+### Shutdown Existing Node
+
+First shutdown both the `js-ceramic` and `kubo` processes using whatever mechanism your infrastructure dictates.
+
+### Migrate IPFS Blocks to Ceramic Events
+
+Kubo stores data as IPFS blocks on disk.
+If you are not using the disk backed Kubo storage first migrate data to an on disk store following Kubo's instructions.
```

Comment:
Could we potentially add a link here for where to find the instructions? I understand this should be somewhere in Kubo's documentation, but having an easy to find link could help following this process easier.

Reply:
Hmm their docs do not seem to exist. I'll reword this